### PR TITLE
Fix translations JS escaping

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
Addresses https://github.com/rapidpro/rapidpro/issues/1221

**Comment by norkans7** [comment]
We can make that approach to many other places we use trans in JS once we agree this is the best solution we can use

File: templates/contacts/contact_list.haml [hunk 1]
```diff
@@ -399,7 +399,7 @@
 
     {% if org_perms.contacts.contact_create %}
     function showCreateContactModal() {
-      var modal = new Modax('{% trans "Create Contact" %}', '{% url "contacts.contact_create" %}')
+      var modal = new Modax('{% trans "Create Contact" as js_modal_title %}{{ js_modal_title|escapejs }}', '{% url "contacts.contact_create" %}')
```
Comment (rowanseymour) [inline templates/contacts/contact_list.haml:402]: I learnt something new about Django templates today.. you can use `_()` in templates https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#string-literals-passed-to-tags-and-filters

So you can simplify the above to `{{ _("Create Contact")|escapejs }}`

Comment (rowanseymour) [reply templates/contacts/contact_list.haml:402]: How did I not know this...

Comment (nicpottier) [reply templates/contacts/contact_list.haml:402]: Neato!

**Pull request merged**